src/components/loading/loading_kibana.test.tsx

```diff
@@ -30,11 +30,15 @@
 
 import React from 'react';
 import { render } from 'enzyme';
-import { requiredProps } from '../../test/required_props';
-
-import { OuiLoadingKibana, SIZES } from './loading_kibana';
+import { requiredProps } from '../../test';
+import { getDeprecatedMessage } from '../../utils';
+import { OuiLoadingKibana, SIZES, WARNING } from './loading_kibana';
 
 describe('OuiLoadingKibana', () => {
+  afterAll(() => {
+    jest.clearAllMocks();
```

Comment:
Why do mocks need to be cleared after each test?

Reply:
Good catch! We don't really need it here